From 202bf1e00eaa533e133c29b509d77cdfb7c13f5e Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Tue, 29 Aug 2017 18:55:37 +0200
Subject: [PATCH 3/3] add more generated files to .gitignore

Make sure all the files generated during a build are properly ignored
from the git status.
---
 .gitignore | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/.gitignore b/.gitignore
index d853d40..87821c3 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,12 @@
+*.a
+*.cma
+*.cmi
+*.cmo
+*.cmx
+*.cmxa
+*.o
+*.so
+*_parser.output
 /_build/
 /setup.data
 /setup.log
@@ -7,6 +16,12 @@
 /configure
 /libgettext-ocaml/META
 /libgettext-ocaml/gettextConfig.ml
+/libgettext-ocaml/gettextPo_parser.mli
 /config.log
 /config.status
+/po/*.mo
 /po/fr.po.bak
+/doc/*.1
+/doc/*.5
+/ocaml-gettext/ocaml-gettext
+/ocaml-gettext/ocaml-xgettext
-- 
2.13.2

